### DISCUSSION TITLE
Where can I enter key and api server when building with cargo run?

### DISCUSSION THREAD

**Discussion by btc2100k** [rustdesk/rustdesk#1487] [Q&A]
Hi, on windows, when I do a custom build of rustdesk via `cargo run`, I am not allowed to enter api server or key for my rustdesk server.

This is causing issue as when a client tries to connect, I get `Failed to receive public key`

<img width="377" alt="Capture" src="https://user-images.githubusercontent.com/5971918/189462827-4f07be3b-7be5-4793-82d6-b59d181e81e4.PNG">

This is what I can enter on the portable 1.1.9 version -- how can I get these things setup in my dev setup?

<img width="381" alt="Capture2" src="https://user-images.githubusercontent.com/5971918/189463273-fb139453-c6e6-4a3e-9c43-c43a68fe597e.PNG">

**Comment by Metathesus** [comment]
Hi,
Seems to be at Line 6x in "\libs\hbb_common\src\config.rs"
